docs/technical/setup-local-dev-env.md

```diff
@@ -0,0 +1,34 @@
+# Setting up local development environment
+
+Creating a service principle
+1. Sign in to your Azure Account through the Azure portal

```

Comment:
As an alternative, you can authenticate via azure CLI, this might be quicker and avoid the need to store secrets, please see here: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/guides/azure_cli

Reply:
This will be really difficult to implement if we want the same terraform configuration to work locally and on the CD pipeline.  Since the CD pipeline will be providing these secrets through variables and we don't want to diverge much from the current terraform configuration, we need to follow suit locally and feed them in the same way.

Price of shared code unfortunately :(

I'll run another idea past you offline that we can explore later on.